recycling-server/test/models/facility_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class FacilityTest < ActiveSupport::TestCase
```

Comment:
You probably want to comment out the test so that CI build could pass since we don't have fixture right now.